### ISSUE TITLE
How could I add the bulb to one of my areas?

### ISSUE THREAD

**Issue by setegonz** [sbidy/wiz_light#20]
Is there an option in the configuration to see my bulb as part of my "Master Bedroom"?

**Comment by sbidy** [comment]
You talking about rooms in the wiz-app context? Than the answer will be -no-. You should select one of your favorite management platform.

I manage my "rooms"/device groups within the HA.

**Comment by setegonz** [comment]
I'm trying to achieve this in the HA configuration.yaml. How can I do it?

**Comment by VerityNH** [comment]
Oh, that's a good adventure you're about to take :-)

So, there might be two ways of doing this,

Do you use automated Lovelace UI layouts or do you configure them manually?

If automated, it should be pretty much straight forward, add an entity in the UI configurator to the desired room tab and that should be it

If you're configuring all the Lovelace layouts manually, you should look into documentation of how to create light groups, Lovelace tabs and so on.

As far as I remember, you're not very experienced in manual configuration, pard me if I'm wrong, so my suggestion here will be to take a look at shared home projects around github. You can use [mine](https://github.com/VerityNH/myhome-hassio) if you'd like to see how things are configured manually.

Basically, your question is not related to something that this integration is intended to support. This is about how you use HA core features. 

Issue suggested to be resolved. 

Thanks
